cmd/kops/create_secret_nodedockerconfig.go

```diff
@@ -0,0 +1,116 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+
+	"github.com/spf13/cobra"
+	"k8s.io/kops/cmd/kops/util"
+	"k8s.io/kops/pkg/apis/kops/registry"
+	"k8s.io/kops/upup/pkg/fi"
+	"k8s.io/kubernetes/pkg/kubectl/cmd/templates"
+	"k8s.io/kubernetes/pkg/util/i18n"
+)
+
+var (
+	create_secret_nodedockerconfig_long = templates.LongDesc(i18n.T(`
+	Create a new node docker config, and store it in the state store. Use update
+	to update it, this command will only create a new entry.`))
+
+	create_secret_nodedockerconfig_example = templates.Examples(i18n.T(`
+	# Create an new node docker config.
+	kops create secret nodedockerconfig -i /path/to/docker/config.json \
```

Comment:
Yeah I guess I wanted to make it clear that it was at the node level and wasn't going to end up inside k8s.  Happy to change it to dockerconfig though.